gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

Comment:
It's not enough to just update the Gradle version here - you need to run `gradle wrapper` to update the wrapper files.

https://docs.gradle.org/8.7/userguide/gradle_wrapper.html#sec:upgrading_wrapper

Reply:
fixed